### PR TITLE
feat: add cue

### PR DESCRIPTION
**Pull request by amaanq**
:8ball:

cc: @eonpatapon @psliwka

### CODE REVIEW

**Review by ObserverOfTime — COMMENTED** [review]
I see some tabs again. :/

File: queries/cue/indents.scm [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+] @indent_end
+
+[ "{" "}" ] @branch
+
+[ "[" "]" ] @branch
+
+[ "(" ")" ] @branch
+
+[
+  (ERROR)
```
Comment (clason) [inline queries/cue/indents.scm:19]: That doesn't look right -- `(ERROR)` is the tree-sitter parsing error node?

**Comment by amaanq** [comment]
> I see some tabs again. :/

This always happens when I cp from the tree-sitter directory to nvim-treesitter...so bizarre, sorry!

File: queries/cue/highlights.scm [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+
+[ "[" "]" ] @punctuation.bracket
+
+[ "(" ")" ] @punctuation.bracket
+
+[ "<" ">" ] @punctuation.bracket
+
+[
+  (ellipsis)
+	"?"
```
Comment (ObserverOfTime) [inline queries/cue/highlights.scm:124]: Missed this one.

```suggestion
  "?"
```

**Review by ObserverOfTime — APPROVED** [review]

**Pull request merged**